app/services/data/get-arms-totals.js

```diff
@@ -5,7 +5,7 @@ const knex = require('knex')(knexConfig)
 module.exports = function (omKey, teamCode) {
   var whereObject = {
     'assessment_staff_key': omKey,
-    'assessmentent_team_key': teamCode // typo in extract
+    'assessment_team_key': teamCode // typo in extract
```

Comment:
can we get get rid of this comment now if the typo has been fixed on this and in extract?

Reply:
Well, we are leaving this as the change is in a migration file and not directly changed in this file.